doc/apiv3-documentation.apib

```diff
@@ -552,17 +572,24 @@ Updates an activity's comment and, on success, returns the updated activity.
 
 # Group Attachments
```

Comment:
No flavour text for attachments? I mean, yes this is pretty basic and obvious stuff, but for the sake of consistency I'd expected some introductory text.